### DISCUSSION TITLE
How to install/use your Pellmon mqtt ?

### DISCUSSION THREAD

**Discussion by lindberg87** [motoz/PellMon#108] [Q&A]
Hey motoz. 
I need your help, with your Pellmon mqtt. 
I have Pellmon system up and running but I want to use the mqtt script you have build. 
How do I add it to the pellmon system ?

Best Regards Stig.

**Comment by motoz** [comment]
PellmonMQTT is a separate project, it communicates with PellMon over the system dbus. There is no installer, so you just copy it somewhere and start it like:

    python pellmonMQTT.py -H yourbroker -d SYSTEM

, where yourbroker is the ip address or hostname to your MQTT broker.

**Comment by lindberg87** [comment]
Thank you very much 👍 😊

Mvh Stig Lindberg


________________________________
Fra: Anders Nylund <notifications@github.com>
Sendt: Wednesday, December 23, 2020 7:05:57 AM
Til: motoz/PellMon <PellMon@noreply.github.com>
Cc: lindberg87 <st1z@live.dk>; Author <author@noreply.github.com>
Emne: Re: [motoz/PellMon] How to install/use your Pellmon mqtt ? (#108)


PellmonMQTT is a separate project, it communicates with PellMon over the system dbus. There is no installer, so you just copy it somewhere and start it like:

python pellmonMQTT.py -H yourbroker -d SYSTEM


, where yourbroker is the ip address or hostname to your MQTT broker.
